packages/transformers/html/src/HTMLTransformer.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+
+import {Transformer} from '@parcel/plugin';
+
+import parse from 'posthtml-parser';
+import render from 'posthtml-render';
+import nullthrows from 'nullthrows';
+import collectDependencies from './dependencies';
+import extractInlineAssets from './inline';
+
+export default new Transformer({
+  canReuseAST(ast) {
+    return ast.type === 'posthtml' && ast.version === '0.0.0';
```

Comment:
use semver.satisfies. Maybe we should make a helper function for this too... e.g. `asset.astSatisfies('posthtml', '^1.2.3')`. (naming tbd)

Reply:
👍I know :) 

I used equality here because of the versioning ambiguity in posthtml, but I'll make the change below and use semver.

re: `astSatisfied`, I'm fine with letting plugins do this on their own, personally.